controllers/release/adapter.go

```diff
@@ -19,14 +19,12 @@ package release
 import (
```

Comment:
I think the changes to this file are a great example of the old function names being better. Checking if a release.isrunning makes sense logically to read. Checking if a release.isreleasing does not. The constant name changes look good, but I don't like the function name changes

Reply:
I can agree for the case of the release, but either we have the wrong name for the CRD itself or we need a better name for the "release" process.

If we understand a Release like a process that involved a preparation, a deployment and the execution of post actions, we can think about a better name for that "preparation".